package.json

```diff
@@ -34,6 +34,9 @@
     "joi": "^14.3.1",
     "yargs": "^11.0.0"
   },
+  "engines": {
```

Comment:
Does adding the `engines` setting actually do anything? As far as I am aware, it is advisory only since `engineStrict` was removed.
https://docs.npmjs.com/files/package.json#engines
https://docs.npmjs.com/files/package.json#enginestrict

That said, I'm happy to leave it in once we confirm the versions to specify.

Reply:
Given this only appears to take effect when this package is installed as a dependency and this is not a library I am happy to omit.

Reply:
It is useful as a guideline to developers but yeah other than that it appears to do nothing unless you're writing a library. Your call :)